README.md

```diff
@@ -95,32 +95,29 @@ If you are interested in making a tauri-app, please visit the [documentation web
 
 | Detail | Tauri | Electron |
 |--|--------|----------|
-| Binary Size MacOS | 0.6 MB (1) | 47.7 MB |
-| Memory Consumption MacOS | 13 MB | 34.1 MB |
+| Installer Size Linux | 3.1 MB | 52.1 MB |
```

Comment:
Did you use this when making your final binary?
```
[profile.release]
panic = "abort"
codegen-units = 1
lto = true
incremental = false
opt-level = "z"
```

Reply:
I used upx when I calculated the MacOS binary size as well...
https://tauri.studio/en/docs/usage/recipes/bundler/anti-bloat#upx

Reply:
I used the cargo config but not upx as that slows down the launch time (right?) and it's not fair to use benchmarks from different apps.

Reply:
It also can't be used on a `.deb` installer.